### PR TITLE
Dislocated jaw changes

### PR DESCRIPTION
**Pull request by printer16**
Speaking with a dislocated jaw causes a drunk-like effect and sometimes a pain message.

Related thread: https://forums.aurorastation.org/viewtopic.php?f=21&p=79658#p79658

### CODE REVIEW

File: code/modules/mob/mob_defines.dm [hunk 1]
```diff
@@ -68,6 +68,7 @@
 	var/ear_damage = null	//Carbon
 	var/stuttering = null	//Carbon
 	var/slurring = null		//Carbon
+	var/brokejaw = null	    //Carbon
```
Comment (Lohikar) [inline code/modules/mob/mob_defines.dm]: Does this need to be all the way up here?

Comment (printer16) [inline code/modules/mob/mob_defines.dm]: It's where all the other speech effects are.

Comment (Lohikar) [inline code/modules/mob/mob_defines.dm]: Do they even need to be up here? They only apply to carbons I believe.

Comment (printer16) [inline code/modules/mob/mob_defines.dm]: I'll move them all down then.

File: code/modules/mob/mob_defines.dm [hunk 2]
```diff
@@ -133,6 +131,10 @@
 	var/obj/item/weapon/storage/s_active = null//Carbon
 	var/obj/item/clothing/mask/wear_mask = null//Carbon
 
+	var/stuttering = null	//Carbon
+	var/slurring = null		//Carbon
```
Comment (Lohikar) [inline code/modules/mob/mob_defines.dm]: I, er, meant move this up to carbon.

Comment (printer16) [inline code/modules/mob/mob_defines.dm]: It appears silicons do need them.

Comment (Lohikar) [inline code/modules/mob/mob_defines.dm]: Alright, update the comment then please.

File: code/modules/mob/living/say.dm [hunk 3]
```diff
@@ -105,7 +105,15 @@ proc/get_radio_key_from_channel(var/channel)
 		message = stutter(message)
 		verb = pick("stammers","stutters")
 		speech_problem_flag = 1
-
+	if(brokejaw)
+		message = slur(message)
+		verb = pick("slobbers","slurs")
+		speech_problem_flag = 1
+		if(prob(50))
+			src << "<span class='danger'>You struggle to speak with a dislocated jaw!</span>"
```
Comment (Lohikar) [inline code/modules/mob/living/say.dm]: Perhaps "You struggle to speak with your dislocated jaw!"?

**Pull request merged**